### ISSUE TITLE
Wrong outputs in TestHardDrive

### ISSUE THREAD

**Issue by bustawin** [eReuse/device-inventory#2]
The following errors in the test field of the hard drive prevents from registering devices (from TxT outputs):

1.  In file 2CE84414SF, 8Q4SR2J, CZC82744TL, CZC82744TT, GDB463J:
   ```json
   {"test":{"status":"required field","type":"required field"}
   ```

   In DeviceHub they are required fields. If indeed they are not for DeviceInventory we can discuss it.
2. In file BDB463J, CFW2Z2J:
    ```json
    {"test":{"type":"unallowed value Short captive"}
    ```

    Please, update the [class diagram](https://drive.draw.io/#G0BzM724dBUg9GNWpiUVJ1LWlCMTQ) with the new types of tests.

**Comment by slamora** [comment]
`type`, `lifetime`, `firstError` are fields related to SMART. If for some reason SMART info could not be retrieved they won't be sent. So, please update that on DeviceHub.

`status` has been fixed on 64fcc4b to be always included.

**Comment by slamora** [comment]
Fixed on e673715.
